src/PaymentGateways/Migrations/CopyV2GatewaysSettingsToV3.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Give\PaymentGateways\Migrations;
+
+use Give\Framework\Migrations\Contracts\Migration;
+
+/**
+ * @unreleased
+ */
+class CopyV2GatewaysSettingsToV3 extends Migration
+{
+
+    /**
+     * @inerhitDoc
+     */
+    public static function id()
+    {
+        return 'copy-v2-gateways-settings-to-v3';
+    }
+
+    /**
+     * @inerhitDoc
+     */
+    public static function timestamp()
+    {
+        return strtotime('2023-09-11');
+    }
+
+    /**
+     * @inerhitDoc
+     */
+    public function run()
+    {
+        $v2Gateways = give_get_option('gateways', []);
+        give_update_option('gateways_v3', $v2Gateways);
```

Comment:
Shouldn't this be filtering out any non v3 compatible gateways?

Reply:
Updated on 68eab7d6bfac871bc38e81d58f430c81d9c74d75